### ISSUE TITLE
Doesn't seem to be paying attention to babelrc?

### ISSUE THREAD

**Issue by probablyup** [developit/microbundle#25]
Fooling around: https://github.com/probablyup/markdown-to-jsx/tree/microbundle

Doesn't seem to be honoring my babelrc and it's blowing up on some ES6+ syntax:

```
yarn run v1.3.2
$ microbundle build
SyntaxError: Unexpected token (553:12) /Users/bear/code/markdown/index.js (nodent)
            ...overrideProps,
------------^
✨  Done in 0.75s.
```

**Comment by developit** [comment]
Indeed - we are actually shipping Bublé right now, not Babel. This is done to produce the smallest possible bundle output, but I think we're going to need to consider either adding a Babel option/plugin (maybe if we detect a babelrc?), or switching to Babel and applying post-transform optimizations to get the size back down to Bublé levels.

**Comment by marvinhagemeister** [comment]
This is resolved with `0.12.0`. We switched from bublé to babel.
